agenta-web/cypress/support/commands/utils.ts

```diff
@@ -1,4 +1,74 @@
+import {randString} from "../../../src/lib/helpers/utils"
+export let app_id
+
 Cypress.Commands.add("clickLinkAndWait", (selector) => {
     cy.get(selector).first().as("link")
     cy.get("@link").click()
 })
+
+Cypress.Commands.add("createVariantsAndTestsets", () => {
+    cy.visit("/settings")
+    cy.get('[data-cy="openai-api-input"]').type(`${Cypress.env("OPENAI_API_KEY")}`)
+    cy.get('[data-cy="openai-api-save"]').click()
+    cy.visit("/apps")
+    cy.get('[data-cy="create-new-app-button"]').click()
+    cy.get('[data-cy="create-from-template"]').click()
+    cy.get('[data-cy="create-app-button"]').click()
+    const appName = randString(5)
+
+    cy.get('[data-cy="enter-app-name-modal"]')
+        .should("exist")
+        .within(() => {
+            cy.get("input").type(appName)
+        })
+
+    cy.get('[data-cy="enter-app-name-modal-button"]').click()
+    cy.intercept("POST", "/api/apps/app_and_variant_from_template/").as("postRequest")
+    cy.wait("@postRequest", {requestTimeout: 15000}).then((interception) => {
+        cy.intercept("GET", `/api/apps/${interception.response.body.app_id}/variants/`).as(
+            "getRequest",
+        )
+        cy.wait("@getRequest", {requestTimeout: 15000})
+
+        cy.url().should("include", `/apps/${interception.response.body.app_id}/playground`)
+        app_id = interception.response.body.app_id
```

Comment:
have you tried this?

`cy.wrap(interception.response.body.app_id).as("app_id")`

If we can alias the app_id, we won't have to save it in an external variable and import in the test cases